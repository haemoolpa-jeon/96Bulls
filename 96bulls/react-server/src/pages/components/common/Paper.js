import React, { Component } from 'react';
import PropTypes from 'prop-types';
import styles from './Paper.module.scss';

class Paper extends Component {
  render() {
    const { children, className } = this.props;
    return (
      <div className={`${styles.paper} ${className}`}>
        {children}
      </div>
    );
  }
}

Paper.propTypes = {
  children: PropTypes.node.isRequired,
  className: PropTypes.string,
};

export default Paper;
