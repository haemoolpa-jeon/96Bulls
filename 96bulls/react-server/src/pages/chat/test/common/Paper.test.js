import React from 'react';
import Paper from '../../components/common/Paper';
import {shallow} from 'enzyme/build';

describe('Paper', () => {
  let component = null;
  it('초기 렌더링이 문제없이 되야함', () => {
    component = shallow(<Paper>
      <p>paper</p>
    </Paper>);
  });

  it('초기 렌더링 스냅샷 일치함', () => {
    expect(component).toMatchSnapshot();
  });
});
