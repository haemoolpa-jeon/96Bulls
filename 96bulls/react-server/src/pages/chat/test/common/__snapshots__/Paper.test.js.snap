// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper 초기 렌더링 스냅샷 일치함 1`] = `
<div
  className="paper undefined"
>
  <p>
    paper
  </p>
</div>
`;
